test/stdlib_checks.dart

```diff
@@ -30,6 +30,10 @@ extension ErrorChecks on Subject<Error> {
   Subject<String> get asString => has((x) => x.toString(), 'toString'); // TODO(checks): what's a good convention for this?
 }
 
+extension StackTraceChecks on Subject<StackTrace> {
+  Subject<String> get asString => has((x) => x.toString(), 'toString'); 
```

Comment:
nit:
```suggestion
  Subject<String> get asString => has((x) => x.toString(), 'toString'); // TODO(checks): what's a good convention for this?
```

(removed trailing space and copied the comment from the other `asString` check)